### ISSUE TITLE
Reddit exeriment

### ISSUE THREAD

**Issue by markchen02** [TUM-DAML/pprgo_pytorch#6]
Table 1 from the paper [Scaling Graph Neural Networks with Approximate PageRank](https://arxiv.org/pdf/2007.01570.pdf) says the accuracy is ~26.6%. The [SOTA](https://paperswithcode.com/sota/node-classification-on-reddit) algorithms seem to have ~97% accuracy. I understand PPRGO is more about scalability, but the difference is quite significant. I wanted to check if the paper using the same Reddit dataset.

Also, do you have the code to evaluate other models in the table (e.g. SGC, Cluster-GCN)? I couldn't find the code in this repo. I would appreciate any information you could provide. Thank you in advance.

**Comment by gasteigerjo** [comment]
It is using the same dataset, but a different dataset _split_. Most papers use a rather extreme number of 153,431 training nodes, i.e. 66%. We instead use a more realistic semi-supervised setup of 820 training nodes, i.e. 0.35%, as described in Section 5.

We have not published the code of baseline models we used for evaluation. Please refer to their reference implementation or to frameworks like PyTorch Geometric for these.
